test/issues/pull.ts

```diff
@@ -0,0 +1,102 @@
+import * as Iridium from "../../iridium";
+import * as chai from "chai";
+
+interface EventType
+{
+    type: string;
+    time: Date;
+};
+
+interface TestDocument {
+    _id?: string;
+
+    stuff?: {
+        [name: string]: {
+            [tracker: number]: EventType[];
+        }
+    };
+}
+
+
+@Iridium.Collection("test")
+@Iridium.Transform(doc => {
```

Comment:
This probably isn't necessary, doesn't look like you're using the `metadata` property here.